src/container/srv_container.c

```diff
@@ -1817,8 +1817,9 @@ ds_cont_tgt_refresh_agg_eph(uuid_t pool_uuid, uuid_t cont_uuid,
 	uuid_copy(arg.cont_uuid, cont_uuid);
 	arg.min_eph = eph;
 
-	rc = dss_task_collective(cont_refresh_vos_agg_eph_one, &arg,
-				 DSS_ULT_FL_PERIODIC);
+	rc = ds_pool_task_collective(pool_uuid, PO_COMP_ST_NEW | PO_COMP_ST_DOWN |
+				     PO_COMP_ST_DOWNOUT, cont_refresh_vos_agg_eph_one,
+				     &arg, DSS_ULT_FL_PERIODIC);
```

Comment:
I think we should add a macro to src/include/daos_srv/pool.h:

```
/* Complement of the map states of targets that make up a pool shard */
#define DS_POOL_TGT_MAP_STATES_EX (PO_COMP_ST_NEW | PO_COMP_ST_DOWN | PO_COMP_ST_DOWNOUT)
```

And use the macro in most cases. I can do this in one of my PRs if you agree.

Reply:
@liw  Sure, please fix this in your PR. Thanks!